presto-tests/src/test/java/io/prestosql/tests/TestDistributedEngineOnlyQueries.java

```diff
@@ -109,23 +109,23 @@ public void testRowTypeWithReservedKeyword()
     @Test
     public void testExplainAnalyze()
     {
-        assertExplainAnalyze(
+        assertExplainAnalyzeSucceeds(
```

Comment:
since these invocations try to validate the provided patterns, `assertExplainAnalyze` or `assertExplainAnalyzeMatches` seems more accurate when `expectedExplainRegExps` are present.  

We could split the usage into two separate methods, and use `assertExplainAnalyzeSucceeds` when no patterns are provided.